### PR TITLE
feat: added Models

### PR DESCRIPTION
**Pull request by mathiasayivor**
I've added the models and we just need to review it for further changes

### CODE REVIEW

File: src/Lib/utils.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+export function isDevMode() {
+    if (process.env.NODE_ENV) {
+        return /^dev(elopment)?$/i.test(process.env.NODE_ENV)
```
Comment (Dun-sin) [inline src/Lib/utils.js:3]: What does this do?

File: src/Models/ShameWhiteListModel.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
+import { model, Schema } from "mongoose";
+
+const ShameWhiteListSchema = new Schema({
```
Comment (Dun-sin) [inline src/Models/ShameWhiteListModel.js:3]: Still don't understand the use of whitelist

File: src/Lib/utils.js (hunk 1 above)
Comment (mathiasayivor) [reply src/Lib/utils.js:3]: It's a helper function to check the current environment i.e development, production

**Review by Dun-sin — APPROVED** [review]

File: src/Models/ShameWhiteListModel.js (hunk 2 above)
Comment (mathiasayivor) [reply src/Models/ShameWhiteListModel.js:3]: It becomes useful in certain edge cases. Check `UserModel.js`

**Pull request merged**